addon/dom-event-listeners.ts

```diff
@@ -1,6 +1,7 @@
 import { assert } from '@ember/debug';
 import { bind } from '@ember/runloop';
-import { registerDisposable } from './utils/disposable';
+import { registerDestructor } from 'ember-destroyable-polyfill';
+import { Destructor } from 'ember-destroyable-polyfill/-internal/destructors';
```

Comment:
Don't think we should need (or use) this private internal interface (it is not in the RFC)...

Reply:
Hmmm ok. I _do_ need to type the second param to `registerDestructor`. Let me see how I can remove it.

Reply:
I removed it.